newscout_web/advertising/urls.py

```diff
@@ -8,19 +8,24 @@
                     CampaignDeleteView, AdGroupView, AdGroupDeleteView,
                     AdvertisementView, AdvertisementDeleteView,
                     CampaignCategoriesListView, GroupTypeListView,
-                    AdRedirectView)
+                    AdRedirectView, AdGroupListView, CampaignListView,
+                    AdvertisementListView)
 
 urlpatterns = [
     url(r'^schedules/$', GetAds.as_view(), name='get-ads'),
     url(r'campaigns/categories/$', CampaignCategoriesListView.as_view(),
         name='campign-categories-view'),
+    url(r'campaigns/list/$', CampaignListView.as_view(), name='campign-list'),
     url(r'campaigns/$', CampaignView.as_view(), name='campign-view'),
     url(r'campaigns/(?P<cid>[-\d]+)/$', CampaignDeleteView.as_view(),
         name='campign-view-delete'),
     url(r'grouptypes/$', GroupTypeListView.as_view(), name='group-type-view'),
+    url(r'adgroups/list/$', AdGroupListView.as_view(), name='adgroup-list'),
     url(r'adgroups/$', AdGroupView.as_view(), name='adgroup-view'),
     url(r'adgroups/(?P<cid>[-\d]+)/$', AdGroupDeleteView.as_view(),
         name='adgroup-view-delete'),
+    url(r'advertisement/list/$', AdvertisementListView.as_view(),
```

Comment:
Make sure to update docs and write tests

Reply:
rushikesh will write updated test in his branch once it is merged.

Reply:
Resolving this, because @rsqwerty will be implementing test and needs this PR to be merged.